applets/clock/clock.c

```diff
@@ -889,6 +896,8 @@ static void
 position_calendar_popup (ClockData *cd)
 {
 #ifdef HAVE_X11
```

Comment:
Ideally this would be `#if defined(HAVE_X11) && defined(GDK_WINDOWING_X11)`
Maybe a bit nitpicky though